### ISSUE TITLE
RDS Migration w/ SSH Tunnel dry-run

### ISSUE THREAD

**Issue by mtrblmoj** [ministryofjustice/cloud-platform#909]
Requirements :

    2 different RDS in different AWS account
    Data in the source DB

Definition of Done :

 - [ ] The data in source DB can now be found in the target DB
 - [ ] The data has been transferred via combo of ssh/pgdump from live-1 to a Template deploy's bastion host.
 - [ ] The steps taken have been documented for use in subesquent data migrations

**Comment by mtrblmoj** [comment]
First Draft : 

0 - have a postgresql/ssh enabled pod running in Live 1
1  - create temp ssh keys to be shared with the pod and the Bastion Host (/home/user/.ssh)
2 - on pod : ssh -L localPort:SourceDataseURL:SourceDatabasePort user@EC2IP  -i privatekey
3 - on pod : pg_dump -U SourceUser -p localPort -d SourceDatabae localhost | psql -U TargetUser -p TargetPort -d TargetDatabaseName

**Comment by mtrblmoj** [comment]
This was tested against PVB, during the migration of their staging RDS from DSD to CP.


A few challenges were faced : 
- User/roles/owners need to be migrated to the TargetDatabase  before the data is transferred.
-- For PVB we created the TargetUser manually 
-- Otherwise we can do: pg_dumpall -g (globals only)
- The SSH tunnel died a few times, while nothing was done, could it fail while we are transferring data ?
- We ended up having an artifact (dump.sql) in the pod, instead of streaming it.
